<?xml version="1.0"?>
<launch>

  <!-- DYNAMIXEL CONTROLLER -->
  <include file="$(find hera_control)/launch/dynamixel_controller.launch"/>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    <rosparam param="source_list" subst_value="true">["dynamixel_controller/joint_states"]</rosparam>
  </node>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/> 

  <include file="$(find hera_moveit_config)/launch/move_group.launch"/>

  <include file="$(find hera_moveit_config)/launch/moveit_rviz.launch">
    <arg name="rviz_config" value="$(find hera_moveit_config)/launch/moveit.rviz"/>
  </include>

  <!-- MANIPULATOR INTERFACE -->
  <node name ="manipulator" pkg="hera_control" type="manipulator.py" output="screen"/>

</launch>
